mitmproxy/tools/web/webaddons.py

```diff
@@ -0,0 +1,18 @@
+class WebOptions:
```

Comment:
how is this consistent?
`consoleaddons.py` contains a `ConsoleAddon` class.
`webaddons.py` contains a `WebOptions` class.